be/test/CMakeLists.txt

```
@@ -67,6 +67,7 @@ set(EXEC_FILES
         ./exec/pipeline/sink/export_sink_operator_test.cpp
         ./exec/pipeline/sink/table_function_table_sink_operator_test.cpp
         ./exec/pipeline/sink/memory_scratch_sink_operator_test.cpp
+        exec/pipeline/exchange_pass_through_test.cpp
```

```suggestion
        ./exec/pipeline/exchange_pass_through_test.cpp
```

Just keep the same style as others although it is not wrong without the `./`